packages/block-library/src/image/edit.js

```diff
@@ -375,8 +375,8 @@ class ImageEdit extends Component {
 			rel,
 			linkClass,
 			linkDestination,
-			width,
-			height,
+			width = 0,
+			height = 0,
```

Comment:
For me undefined width/height is different in meaning from a width=0 and height=0 and the checks that were changed line 484 illustrates this. 

I think a safer fix would be to avoid adding default values here and instead treat the symptom (add a check to the inputs triggering the warning) `width || ""`, `height || ""` (I don't think we want to show 0 in these inputs when the value is undefined)

Reply:
Hmm that would work for me too. No strong opinion though.

Reply:
Hi @youknowriad, thanks for your suggestion.

How does https://github.com/WordPress/gutenberg/pull/15239/commits/282e052b40bd61700c4a5929d0a83989123678f7 look? I'm not sure if it captures your suggestion.

There's a failed [E2E test](https://travis-ci.com/WordPress/gutenberg/jobs/199450362) that I could fix.

Reply:
Yep, that's exactly what I suggested, thanks.

The test failure is unrelated to this PR (restarted and passed) and is being worked on separately.